docs/projects/README.md

```diff
@@ -0,0 +1,469 @@
+requirements:
+
+- The issue follows the
+  [project thread issue template](templates/project-thread-issue-template.md)
+- The `project` label is applied to the issue
+- A `status` is applied to the issue via our GitHub Project Board
+
+These threads are for the benefit of our contributors, but also external
+stakeholders to the project. They provide visibility into our current
+initiatives and up to date information on each initiative. Each project thread
+is a place to see the current status, timeline, and motivation of a given work
+stream.
+
+All requirements for working with project threads can be found in this document.
+If you have any questions please feel free to reach out to an Openverse
+maintainer in the #openverse channel of the
+[Making WordPress Chat](https://make.wordpress.org/chat) or
+[file an issue](https://github.com/WordPress/openverse/issues/new/choose).
+
+## Where documents are kept, how they are named, and how they are reviewed
+
+For each project, planning documents should be kept in the `rfcs/` directory of
+this repository. Each project should have its own subdirectory. Projects
+predating this document have been updated to follow this structure. Individual
+documents should be date-stamped in the `YYYYMMDD` format. An example of this
+entire structure follows:
+
+```
+| rfcs/
+   | service_metrics/
+      - YYYYMMDD-project_proposal.md
+      - YYYYMMDD-implementation_plan.md
+   | 3d_model_support/
+      - YYYYMMDD-project_proposal.md
+      - YYYYMMDD-implementation_plan_(catalogue).md
+      - YYYYMMDD-implementation_plan_(api).md
+      - YYYYMMDD-implementation_plan_(frontend).md
+```
+
+Using subdirectories makes it slightly easier to navigate an ever-growing list
+of planning documents. This is especially the case in projects similar to the 3D
+Models project in the example above that may require multiple implementation
+planning documents for each part of the application. In the process of planning
+or implementing some projects, we may develop additional supporting documents or
+appendixes that go beyond the required scope of the project planning process
+described in this document. Those documents should accompany the required ones
+in the same subdirectory for the project.
+
+All project documentation can be authored with any tool, but must be written in
+[GitHub Flavored Markdown](https://github.github.com/gfm/) including the
+available
+[GitHub-specific advanced formatting features](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting).
+
+Additionally, all documents should be formatted to pass the Prettier linting
+step required by the repository for Markdown documents.
+
+When submitting a new document for a project, the author should open a pull
+request with the document. The review process of the document will happen as a
+PR review. Once the document is accepted and approved, it is merged and the
```

Comment:
Just to confirm, this PRs will also require 2 approvals, just as our usual PRs?

Reply:
Yep, that's correct.